heudiconv/bids.py

```diff
@@ -487,12 +486,34 @@ def convert_sid_bids(subject_id):
     subject_id : string
         Original subject ID
     """
+    sid, subject_id = sanitize_label(subject_id)
+    lgr.warning('Deprecation warning: convert_sid_bids() is deprecated, '
+                'please use sanitize_label() instead.')
+    
+    return sid, subject_id
```

Comment:
just issue warning before even calling and then here

```
return sanitize_label(subject_id)
```
since there is no need to assign those to any variable meanwhile right?